com.github.linthesia.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "com.github.linthesia",
```

Comment:
the app-id is invalid: it is missing the application part: it should be `com.github.linthesia.linthesia`.